### PR TITLE
feat: Select组件添加renderTag实现自由定制tag内容, 添加操作插槽

### PR DESCRIPTION
**Pull request by wanchun**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [x] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**Related issue (if exists):**

**Other information:**

### CODE REVIEW

File: docs/.vitepress/components/select/index.md [hunk 1]
```diff
@@ -120,11 +123,12 @@ app.use(FSelect);
...
-| ------- | --------------- |
-| default | option 组件列表 |
-| empty   | 无选项的内容    |
-| option   | 自定义 `Option` 内容    |
+| 名称    | 说明            | 参数 |
+| ------- | --------------- | --------------- |
+| default | option 组件列表 | -  |
+| empty   | 无选项的内容    | - |
+| option   | 自定义 `Option` 内容    | *{ value, label, disabled, isSelected }* |
+| tag   | 自定义 `Tag` 内容    | *{ option: Option, handleClose: ()=> void }*  |
```
Comment (winixt) [inline docs/.vitepress/components/select/index.md:131]: 缺少 action  slot 的文档描述

Comment (winixt) [reply docs/.vitepress/components/select/index.md:131]: 对于 select 来说，什么是 tag ? 指：选中的值？

File: components/select/style/index.less [hunk 2]
```diff
@@ -32,6 +32,11 @@
             height: @select-dropdown-max-height;
         }
     }
+    &-action {
+        .text();
+        padding: @padding-xs;
+        border-top: 1px solid var(--f-border-color-split);
```
Comment (winixt) [inline components/select/style/index.less:38]: 1px为什么没有用变量？

File: components/select/select.vue [hunk 3]
```diff
@@ -230,17 +238,9 @@ export default defineComponent({
...
-                                        ...option,
-                                        label: option[props.optionLabelField],
-                                    };
-                                }
-                                return option;
-                            })
-                            .find((option) => option.value === val);
+                        cacheOption = newOptions.find(
+                            (option) => option.value === val,
+                        );
```
Comment (winixt) [inline components/select/select.vue:243]: 这块逻辑有点啰嗦，const cacheOption = [...(newOptions || []), ...selectedOptionsRef.value].find(xxx)

File: components/select-tree/selectTree.vue [hunk 4]
```diff
@@ -292,22 +293,12 @@ export default defineComponent({
...
-                            label: option[props.optionLabelField],
-                        };
-                    }
-                    return option;
-                }),
+            Object.values(nodeList.value).filter((option) => {
+                if (props.multiple) {
+                    return currentValue.value.includes(option.value);
+                }
+                return [currentValue.value].includes(option.value);
```
Comment (winixt) [inline components/select-tree/selectTree.vue:300]: 搞个 === 多好

File: components/select-trigger/renderTag.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+export default defineComponent({
+    props: {
+        option: Object as PropType<SelectOption>,
+        renderTag: Function as PropType<(param: RenderTagParam) => VNodeChild>
+    },
+    emits: ['close'],
+    setup(props, {emit, slots}) {
+        return () => {
+            const { option, renderTag } = props;
+            const handleClose = ()=>{
```
Comment (winixt) [inline components/select-trigger/renderTag.tsx:15]: 函数定义放在 return 外面好一些，不然每次 render 都得重新初始化

File: components/select/select.vue (hunk 3 above)
Comment (wanchun) [reply components/select/select.vue:243]: 性能好点，条理也清楚，不改

**Pull request merged**